src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+$blueColor: #00acdc;
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $cardWidth;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    width: 100%;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 0;
+    color: #060b35;
+  }
+
+  &__product {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 4px 0 0;
+    width: 100%;
+    color: #616070;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 0;
+    width: 100%;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 18px;
+    line-height: 18px;
+    margin: 24px 0 0;
+    width: 100%;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__link {
+    display: block;
+    justify-content: center;
+    background-color: $blueColor;
+    height: 40px;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #f3f3f3;
+    margin-top: 16px;
+    width: 100%;
+    bottom: 1px;
```

Comment:
The `bottom` property is used here, but it will have no effect unless the element is positioned (e.g., `position: relative;`, `absolute;`, `fixed;`, or `sticky;`). Consider adding a positioning property if needed.